mypy.ini

```diff
@@ -1,4 +1,5 @@
 [mypy]
```

Comment:
We could get better mypy error messages using these options: `--show-error-context --show-error-codes --pretty`